wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/AstToTaskBodyElement.scala

```diff
@@ -0,0 +1,14 @@
+package wdl.draft3.transforms.ast2wdlom
+
+import common.Checked
+import common.validation.Checked._
+import wdl.draft3.parser.WdlParser.Ast
+import wdl.model.draft3.elements.TaskBodyElement
+
+object AstToTaskBodyElement {
+  def convert(ast: Ast): Checked[TaskBodyElement] = ast.getName match {
+    case "Inputs" => astNodeToInputsSectionElement(ast)
+    case "Outputs" => astNodeToOutputsSectionElement(ast)
+    case other => s"No conversion defined for Ast with name $other".invalidNelCheck
```

Comment:
I'd be more specific here to help with debugging if we make mistakes:
`"No conversion defined from Ast with name '$other' to TaskBodyElement"`